runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/JavaReadViaImpulseTest.java

```diff
@@ -106,6 +149,13 @@ public void testOutputCoder() {
         equalTo(BigEndianIntegerCoder.of()));
   }
 
+  private static void assertNotReadTransform(PTransform<?, ?> transform) {
```

Comment:
This could be inlined into the anonymous subclass that you'er traversing the pipeline with

Reply:
As above, this is already submitted, though I could change it here if you want.